docs/root/intro/arch_overview/security/ssl.rst

```diff
@@ -182,6 +182,30 @@ infrastructure.
 Client TLS authentication filter :ref:`configuration reference
 <config_network_filters_client_ssl_auth>`.
 
+.. _arch_overview_ssl_custom_handshaker:
+
+Custom handshaker extension
+---------------------------
+
+The :ref:`CommonTlsContext <envoy_v3_api_msg_extensions.transport_sockets.tls.v3.CommonTlsContext>`
```

Comment:
Can you ref link directly to the extension field?

Reply:
Done.